### DISCUSSION TITLE
Setting up WSA on Surface Pro running Win11 Pro on ARM

### DISCUSSION THREAD

**Discussion by greatg67** [MustardChef/WSABuilds#469] [Q&A]
Hello,

Been trying to set up the WSA for ARM with very little success. I have followed the instructions and after the Powershell is closed, 2 instances of WSA open up. They stay open for about 5min and then both close? Magisk doesn't appear to install and playstore doesn't run? Any help would be great. I did try the X86 version and it worked on my desktop!?

**Comment by MustardChef** [comment]
@greatg67 
Get WSA from: https://github.com/snickler/WSA-Patched/releases/tag/0.0.2,

**Reply by greatg67** [reply]
I followed instructions from XDA that used this patch. However, still can't install Google Playstore. Tried various apk files from APKMirror and they would no install.
